tests/migration_test.go

```diff
@@ -246,7 +246,6 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			withKernelBoot(),
 			withSecret(secretName),
 			withConfigMap(configMapName),
-			libvmi.WithCloudInitNoCloudUserData("#!/bin/bash\necho 'hello'\n", true),
```

Comment:
This has to stay as we are trying to verify we can migrate VM with cloudinit

Reply:
oh, what a shame. You are so very right. Sorry for the noise.